src/table.ts

```diff
@@ -43,25 +44,30 @@ export type CreateTableCallback = LongRunningCallback<Table>;
 export type DropTableResponse = UpdateSchemaResponse;
 export type DropTableCallback = UpdateSchemaCallback;
 
+interface MutateRowsOptions extends CommitOptions {
+  requestOptions?: Omit<IRequestOptions, 'requestTag'>;
+}
+
 export type DeleteRowsCallback = CommitCallback;
 export type DeleteRowsResponse = CommitResponse;
-export type DeleteRowsOptions = CommitOptions;
+export type DeleteRowsOptions = MutateRowsOptions;
 
 export type InsertRowsCallback = CommitCallback;
 export type InsertRowsResponse = CommitResponse;
-export type InsertRowsOptions = CommitOptions;
+export type InsertRowsOptions = MutateRowsOptions;
 
 export type ReplaceRowsCallback = CommitCallback;
 export type ReplaceRowsResponse = CommitResponse;
-export type ReplaceRowsOptions = CommitOptions;
+export type ReplaceRowsOptions = MutateRowsOptions;
 
 export type UpdateRowsCallback = CommitCallback;
 export type UpdateRowsResponse = CommitResponse;
-export type UpdateRowsOptions = CommitOptions;
+export type UpdateRowsOptions = MutateRowsOptions;
 
 export type UpsertRowsCallback = CommitCallback;
 export type UpsertRowsResponse = CommitResponse;
-export type UpsertRowsOptions = CommitOptions;
+export type UpsertRowsOptions = MutateRowsOptions;
+
```

Comment:
I think `table.createReadStream()` and `table.read()` below should support setting a request tag.

Reply:
I believe it does. `requestTag` is included in the `ReadRequest` interface that is used with both `table.createReadStream()` and `table.read()`